solana/solana-ibc/programs/solana-ibc/src/lib.rs

```diff
@@ -448,6 +530,39 @@ pub struct SendPacket<'info> {
     system_program: Program<'info, System>,
 }
 
+impl<'a> Deliver<'a> {
+    fn to_transfer_accounts(&self) -> TransferAccounts<'a> {
+        TransferAccounts {
+            sender: Some(self.sender.as_ref().to_account_info()),
+            receiver: self
+                .receiver
+                .clone()
+                .map(|account| account.to_account_info()),
```

Comment:
Alternatively:
```suggestion
                .map(ToAccountInfo::to_account_info),
```
Matter of taste I suppose.

Reply:
But i was able to do the above for accounts which are not boxed ( `receiver` ).

Reply:
Nvm it worked with `deref` instead.